apps/codeforafrica/src/components/OurPartners/OurPartners.js

```diff
@@ -61,6 +63,22 @@ const OurPartners = React.forwardRef(function OurPartners(props, ref) {
           );
         })}
       </Grid>
+      {action && (
+        <Button
+          variant="contained"
+          component={Link}
+          href={action?.href}
+          sx={{
+            display: "block",
+            mt: 7.25,
+            mx: "auto",
+            textAlign: "center",
+            width: { xs: "100%", sm: "fit-content" },
+          }}
+        >
+          {action?.content || action?.href}
+        </Button>
+      )}
```

Comment:
```suggestion
      {action?.href && (
        <Button
          variant="contained"
          component={Link}
          href={action.href}
          sx={{
            display: "block",
            mt: 7.25,
            mx: "auto",
            textAlign: "center",
            width: { xs: "100%", sm: "fit-content" },
          }}
        >
          {action?.content || action.href}
        </Button>
      )}
```

Best to straight up check for existence of `action.href`